CONTRIBUTING.md

```diff
@@ -20,8 +20,23 @@ Simple Icons welcomes contributions and corrections. Before contributing, please
 We welcome icon requests. Before you submit a new issue please make sure the icon:
 
 * Has not already been requested. If you find an existing issue or pull request for the brand you're looking for then please add a reaction or comment to show your support.
-* Is of a _popular_ brand. For websites, the [Alexa rank](https://www.alexa.com/siteinfo) should be less than 500k. For anything else, popularity will be judged on a case-by-case basis.
-* Isn't related to anything that provides an illegal service (e.g. piracy, malware, threatening material, spam, etc.).
+* Is of a _popular_ brand:
+    - For websites, the [Alexa rank](https://www.alexa.com/siteinfo) should be less than 500k.
+    - For GitHub projects, the amount of "stars" should be above 5k.
+    - For anything else, popularity will be judged on a case-by-case basis.
+* Doesn't fall into one of the following categories:
+    - Anything that provides an illegal service (e.g. piracy, malware, threatening material, spam etc.)
+    - Governmental agencies, programs, departments.
+       - Allowed: International organizations and NGOs with supranational interests (e.g. WHO and Unicef).
+       - Allowed: Organizations like NASA or ESA.
```

Comment:
These ends with dots... should be removed to maintain the consistency?

Reply:
Sorry. 😅  I'll remove them.